lib/mayaUsd/render/MaterialXGenOgsXml/libraries/mx_lighting_maya_none.glsl

```diff
@@ -10,10 +10,10 @@
 
 vec3 mx_environment_irradiance(vec3 N)
 {
-    return vec3(0);
+    return vec3(0.0);
 }
 
-vec3 mx_environment_radiance(vec3 N, vec3 V, vec3 X, vec2 alpha, int distribution, FresnelData fd)
+vec3 mx_environment_radiance(vec3 N, vec3 V, vec3 X, vec2 roughness, int distribution, FresnelData fd)
 {
-    return vec3(0);
+    return vec3(0.0);
 }
```

Comment:
Took the latest [mx_environment_none.glsl](https://github.com/AcademySoftwareFoundation/MaterialX/blob/main/libraries/pbrlib/genglsl/lib/mx_environment_none.glsl) from MaterialX.